platform/core/src/capabilities/PersistenceCapability.js

```diff
@@ -148,13 +148,19 @@ define(
          */
         PersistenceCapability.prototype.refresh = function () {
             var domainObject = this.domainObject;
+            var $q = this.$q;
 
             // Update a domain object's model upon refresh
             function updateModel(model) {
-                var modified = model.modified;
-                return domainObject.useCapability("mutation", function () {
-                    return model;
-                }, modified);
+                if (model === undefined) {
```

Comment:
Okay not to address the heart of the issue as we have a sprint dedicated to refactoring the persistence stack. Tracked https://github.com/nasa/openmct/issues/1545 for the modified timestamp.